ui/src/pages/research-culture-report.tsx

```diff
@@ -0,0 +1,162 @@
+                    <Components.PageTitle
+                        className="mb-8"
+                        text={`New report questions impact of
+publishing on research culture`}
+                    />
+                    <h2 className="flex space-x-1 pb-16 font-montserrat text-xl font-normal">
+                        Open research publishing platforms could be an important mechanism for reform
+                    </h2>
+                    <div className="text-sm text-grey-800 transition-colors duration-500 dark:text-grey-100">
+                        <p>
+                            To explore the need for a platform like Octopus and create a benchmark for its
+                            effectiveness, a report was conducted by researchers from the University of Bristol, which
+                            found academic researchers demoralised by a culture that disincentivises sharing and
+                            collaboration, encourages questionable research practices (QRPs), and increases the risk of
+                            bias.
+                        </p>
+                        <br />
+                        <p>
+                            The researchers carried out a literature study, one-to-one interviews, and a survey of over
+                            400 research professionals to understand how they felt about their work, careers, and the
+                            state of open research.
+                        </p>
+                        <br />
+                        <p>Their findings are published in a report which can be accessed here:</p>
+                        <br />
+                        <br />
+                        <p className="text-center">
+                            <Components.Link
+                                className="rounded-lg border-transparent bg-grey-700 p-4 font-medium text-white-50 decoration-teal-500 underline-offset-2 outline-0 transition-colors duration-500 hover:bg-grey-600 focus:overflow-hidden focus:ring-2 focus:ring-yellow-400 dark:bg-teal-600 dark:hover:bg-teal-600 sm:w-fit sm:px-4"
+                                title="Report"
+                                href="https://doi.org/10.5281/zenodo.8165704"
+                                openNew
+                            >
+                                Read the report
+                            </Components.Link>
+                        </p>
+                    </div>
+                    <br />
+                    <br />
+                    <h3 className="font-montserrat text-xl font-semibold">Summary:</h3>
+                    <br />
+                    <div className="text-sm text-grey-800 transition-colors duration-500 dark:text-grey-100">
+                        <p>
+                            The work, “A snapshot of the academic research culture in 2023 and how it might be
```

Comment:
hopefully these quotes (and the single ones) will render fine on all browsers... personally I'd swap for standard double/single quotes

Reply:
replaced all quotes (single and double) with html codes